src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -39,6 +39,7 @@ namespace Microsoft.Azure.WebJobs.Script
     public class ScriptHost : JobHost, IScriptJobHost
     {
         internal const int DebugModeTimeoutMinutes = 15;
+        private const int DelayedFileWatchersDelaySeconds = 30;
```

Comment:
This isn't really a file watcher delay, it's a background purge delay. Rename to FileSystemCleanupDelay?

Reply:
Ah, forgot to rename this. Thanks